images/quayio-pull-through-cache/docker-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+secret_path=/etc/pull-secret/.dockerconfigjson
+ls -l -S $secret_path 2>/dev/null || echo "Missing $secret_path" && exit 1
+export REGISTRY_PROXY_USERNAME=$(cat $secret_path | jq '.auths["quay.io"].email' | cut -d '"' -f 2)
```

Comment:
The the case of our service account, I think username differs from email. Username looks like `openshift-release-dev+ocm_access_7bf7....` and email looks like `aos-team-dp-testplatform@redhat.com` in the secret. Docker buries username in the `auth` field.
`jq '.auths["quay.io"].auth' -r | base64 -d`  will show you the username/password pair. They are colon delimited.